src/app/components/crypto-assets/bitcoin/brc20-token-asset-list/components/brc20-token-asset-item.layout.tsx

```diff
@@ -1,58 +1,59 @@
-import { BoxProps, Flex } from '@stacks/ui';
-import { forwardRefWithAs } from '@stacks/ui-core';
-import { HStack, styled } from 'leather-styles/jsx';
+import { BoxProps, Flex, HStack, styled } from 'leather-styles/jsx';
 
 import type { Money } from '@shared/models/money.model';
 
 import { formatBalance } from '@app/common/format-balance';
 import { AssetCaption } from '@app/components/crypto-assets/components/asset-caption';
-import { Brc20TokenIcon } from '@app/components/icons/brc20-token-icon';
 import { usePressable } from '@app/components/item-hover';
 import { Flag } from '@app/components/layout/flag';
 import { Tooltip } from '@app/components/tooltip';
+import { Brc20TokenIcon } from '@app/ui/components/icons/brc20-token-icon';
 
 interface Brc20TokenAssetItemLayoutProps extends BoxProps {
   balance: Money;
   caption: string;
   isPressable?: boolean;
   title: string;
 }
-export const Brc20TokenAssetItemLayout = forwardRefWithAs(
-  (props: Brc20TokenAssetItemLayoutProps, ref) => {
-    const { balance, caption, isPressable, title, ...rest } = props;
-    const [component, bind] = usePressable(isPressable);
+export function Brc20TokenAssetItemLayout({
+  balance,
+  caption,
+  isPressable,
+  title,
+}: Brc20TokenAssetItemLayoutProps) {
+  const [component, bind] = usePressable(isPressable);
 
-    const formattedBalance = formatBalance(balance.amount.toString());
+  const formattedBalance = formatBalance(balance.amount.toString());
 
-    return (
-      <Flex as={isPressable ? 'button' : 'div'} outline={0} ref={ref} {...rest} {...(bind as any)}>
-        <Flag align="middle" img={<Brc20TokenIcon />} spacing="base" width="100%">
-          <HStack alignItems="center" justifyContent="space-between" width="100%">
-            <styled.span
-              maxWidth="150px"
-              overflow="hidden"
-              textAlign="left"
-              textOverflow="ellipsis"
-              textStyle="label.01"
-              whiteSpace="nowrap"
-            >
-              {title}
+  return (
+    // #4476 TODO not sure why this isn't erroring
+    <Flex as={isPressable ? 'button' : 'div'} outline={0} {...(bind as any)}>
```

Comment:
because of the any?

Reply:
I think `<Flex as=` should have an error now as it shouldn't be OK with `leather-styles/jsx`

<img width="194" alt="Screenshot 2023-11-17 at 11 02 43" src="https://github.com/leather-wallet/extension/assets/2938440/a6c0068b-8f82-4345-bd91-5ec3946eb426">